src/main/java/com/zenjava/javafx/maven/plugin/NativeMojo.java

```diff
@@ -185,6 +185,22 @@
      */
     protected String appName;
 
+    /**
+     * Set this to true if you would like to install your application in the system directory (i.e. Program Files on
+     * Windows) instead of a user specific one. Note that when you activate this option, you need admin privilege to
+     * install your application.
+     *
+     * @parameter default-value=false
+     */
+    protected boolean installForAllUsers;
+
+    /**
+     * The icon file to customize icon application launcher. On Windows, the .ico format is expected, and on Mac it is
+     * .icns. No icon is embedded into launcher on Linux.
+     *
+     * @parameter
+     */
+    protected File appIcon;

```

Comment:
> just because the path you've given is dependent of the app name
@mfalaize this does not come from us, its the template from WiX/InnoSetup, but you are right, its not just undocumented.

@maxd We may have to check/rework all parts for customizing the launcher itself.

Let me tinker something the next days, im sure we will come up with a good solution, but i dont feel good about this PR right now.